what's the pim marketplace article (#973)
akeneo/pim-helpcenter: content/md/what-is/what-is-the-pim-marketplace.md

:::

Thanks to our community and partners, we have plenty of extensions ready to be connected to your PIM. 
From Import/Export to e-commerce, including Translation, DAM, Print, or Syndication extensions, this is a whole new world that opens to you!

But how to find these extensions, how to get in touch with their creators, how to buy a connector? Lots of questions to answer! That's why we created the `Akeneo marketplace` you can access right here: [https://marketplace.akeneo.com/](https://marketplace.akeneo.com/)

# What's the PIM Marketplace?

To ease the way to find the right extension to connect your Akeneo PIM to its ecosystem, we developed the PIM `Marketplace`. 

![PIM Marketplace](../img/Pim-marketplace.png)

Thanks to the PIM `Marketplace`, you can see all the connectors available on our Marketplace and compatible with your Akeneo PIM version and edition. When you find a connector that could be interesting, click on `More info` to reach the connector documentation. 

From that page, you'll be able to read all the connector documentation, consult all the available screenshots, reach the connector developer or even download it. 

> To find the connectors you may need, go to `Connect`, then click on `Marketplace`, and here you are! 

Reviewer: ```suggestion
To find the connectors you need, go to `Connect`, then click on `Marketplace`, and here you are! 
```

# What's a connector?
A connector is a bridge between Akeneo PIM and a third-party component or application. 

> It enables to collect data or spread it to one or several channels. 

Reviewer: ```suggestion
It enables data collection and sharing to one or several channels. 
```

> To comply with the Serenity Edition (full-SaaS mode), Akeneo connectors should rely on our web API and be hosted outside the PIM interface. Akeneo API provides all necessary endpoints to connect the PIM structure and entities with any application.

Reviewer: ```suggestion
```
